flow/core/params.py

```diff
@@ -395,6 +395,7 @@ class SimParams(object):
 
     def __init__(self,
                  sim_step=0.1,
+                 scale=1.0,
```

Comment:
Hi, could you add an explanation of what this does to the docstring?

Reply:
Sure thing, done :)